src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+                    break;
+                case (2):
+                    outputSplitLine();
+                    outputDataInConsole("For find hotel by city complete the following form");
+                    outputDataInConsole(findHotelByCityMenu().toString());
+                    outputSplitLine();
+                    break;
+                case (3):
+                    outputSplitLine();
+                    outputDataInConsole("To find the room fill in the following form");
+                    findRoomMenu();
+                    outputSplitLine();
+                    break;
+                case (4):
+                    outputSplitLine();
+                    outputDataInConsole("Your reserved rooms: ");
+                    getAllUserReservationsMenu();
+                    break;
+                case (5):
+                    outputSplitLine();
+                    outputDataInConsole("To cancel a reservation of the room, fill in the following form");
+                    cancelReservationMenu();
+                    outputSplitLine();
+                    break;
+                case (6):
+                    outputSplitLine();
+                    outputDataInConsole("\tThank you for using our service.");
+                    outputSplitLine();
+                    flagMenu = false;
+                    logoutUserMenu();
+                    continue;
+                default:
+                    outputDataInConsole("Incorrect menu item");
+                    serviceMenu();
+                    continue;
+            }
+        }
+    }
+
+    private void registrationUserMenu() {
+        String firstName = null;
+        String lastName = null;
+        outputSplitLine();
+        try {
+            firstName = String.valueOf(inputDataFromUser("Enter name: ", false,
+                    false, true, false));
+            lastName = String.valueOf(inputDataFromUser("Enter last name: ", false,
+                    false, true, false));
+            userController.loginUser(firstName, lastName);
+        } catch (IOException e) {
+            outputDataInConsole("Is invalid user data");
+        }
+        outputDataInConsole(firstName + ", You have successfully logged in!\n\n\t Welcome to our site!\n");
+    }
+
+    private void logoutUserMenu() throws IOException {
+        userController.logoutUser();
+        mainMenu();
+    }
+
+    private List<Hotel> findHotelByNameMenu() {
+        String hotelName = null;
+        try {
+            hotelName = String.valueOf(inputDataFromUser("Enter hotel name (to exit enter \"0\"): ",
+                    false, false, true, false));
+        } catch (IOException e) {
+            outputDataInConsole("Entered incorrect name of the hotel");
+        }
+        if (exitMenu(hotelName)) {
+            try {
+                serviceMenu();
+            } catch (IOException e) {
+                outputDataInConsole("Don't loaded Service menu" + e);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Don't loaded Service menu" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)